src/main/java/trade/wayruha/mexc/dto/Trade.java

```diff
@@ -0,0 +1,91 @@
+package trade.wayruha.mexc.dto;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Data;
+import trade.wayruha.mexc.enums.OrderSide;
+
+import java.math.BigDecimal;
+
+import static java.util.Objects.isNull;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Trade {
+
+    private String tradeId;
+
+    @JsonAlias({"i"})
+    private String orderId;
+
+    @JsonAlias({"c"})
+    private String clientOrderId;
+
+    private Long orderListId;
+
+    private OrderSide side;
+
+    @JsonProperty("S")
+    public void setOrderSideByAlias(int aliasNumber) {
+        this.side = OrderSide.findByAlias(aliasNumber);
+    }
```

Comment:
@JsonProperty on a setter VS @jsonValue inside enum?

Reply:
**The same problem here** 
`
WebSocket and REST send different type of values (WS send numbers, REST send string). This kind of using JsonProperty resolve this in simply way, creation of custom deserializer was the longest.`